.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,22 @@
+---
+name: issue_template
+about: Informs contributors of how issues will be reviewed
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Thank you for submitting this issue to AMY.  If this represents an urgent problem such as AMY not loading at all, please contact team@carpentries.org so we can respond promptly.
+
+All other issues will be reviewed on a weekly basis by @maneesha.  She will follow up here with you and/or other team members if more information is needed. Issues will be prioritized, considering whether they fix broken features, enhance existing features, or implement entirely new features.
```

Comment:
"All other" --> "Non-urgent"
"on a weekly basis" --> "weekly"

Reply:
"fix broken features . . . " --> "identify broken features, identify potential enhancements to existing features, or request implementation of entirely new features" (because there are issues, not PRs)